doc/4-instrument/es5503.md

```diff
@@ -0,0 +1,45 @@
+# ES5503 instrument editor
+
+ES5503 instrument editor is divided into 4 tabs:
+
+- **ES5503**: for controlling the parameters of ES5503 sound channel.
+- **Wavetable**: for wavetable-related controls.
+- **Macros**: for macros controlling volume/arp/pitch/panning/etc.
+- **Sample**: for sample-related controls.
+
+## ES5503
+
+There is a tricky feature of ES5503 when it comes to playing wavetables/samples. ES5503 can play waves of sizes 256, 512, ..., 32768 steps. When it encounters `0` in sample memory it stops playing the wavetable/sample, no matter what oscillator mode and wave size are.
+
+- **Oscillator mode**: lets you select one of four possible oscillator modes:
+  - **Freerun**: oscillator plays the whole wavetable/sample and loops back if it haven't encountered `0` in sample memory along the way. So if sample does not exactly fit to the one of aforementioned possible sizes it will stop playing after being played once.
+  - **Oneshot**: oscillator plays the whole wavetable/sample once.
+  - **Sync/AM**: sync mode is only possible between pairs of even and odd numbered oscillators, i.e. a lower even-numbered oscillator paired with a higher odd-numbered oscillator. When the even-numbered oscillator starts playing back its wavetable, its odd-numbered mate will be synchronized to it and begins its wavetable/sample simultaneously (aka hard sync). (shamelessly stolen from [here](http://www.buchty.net/ensoniq/5503.html)) AM means amplitude modulation and works in reverse - odd-numbered oscillator modulates the volume of previous even-numbered oscillator. It's kinda tricky to work with this mode.
```

Comment:
```suggestion
  - **Sync/AM**: sync mode is only possible between pairs of even and odd numbered oscillators, i.e. a lower even-numbered oscillator paired with a higher odd-numbered oscillator. When the even-numbered oscillator starts playing back its wavetable, its odd-numbered mate will be synchronized to it and begins its wavetable/sample simultaneously, aka hard sync. (info from [www.buchty.net](http://www.buchty.net/ensoniq/5503.html) AM means amplitude modulation and works in reverse; odd-numbered oscillator modulates the volume of previous even-numbered oscillator. It's kinda tricky to work with this mode.
```

I don't want to strip the personality from everything, but "shamelessly stolen" might be a bit much. :)